### ISSUE TITLE
redis host config get error

### ISSUE THREAD

**Issue by springjk** [ankitpokhrel/tus-php#70]
In my laravel case, `REDIS_HOST` is not `127.0.0.1`, it config in laravel `.env` file, if i use the clear command bin like reademe : `./vendor/bin/tus tus:expired redis`, it will get an error

```
In AbstractConnection.php line 155:

  Connection refused [tcp://127.0.0.1:6379]
```

https://github.com/ankitpokhrel/tus-php/blob/master/src/Cache/CacheFactory.php#L20
because there can't get the `REDIS_HOST`  env value, `./vendor/bin/tus tus:expired redis` don't run via laravel framework, so laravel config environment does't exist, my solution is make it run in  laravel framework， add to laravel schedule job, like this：

```
app/Console/Kernel.php

protected function schedule(Schedule $schedule)
{
    $schedule->exec('./vendor/bin/tus tus:expired redis')->dailyAt('01:00');
}
```

I think you can update readme to help other people or do a better solution. 😎

**Comment by springjk** [comment]
btw, redis port also can use this solution，but password can't，there have no code to get password, even i can copy or overwrite a new command to do it，but i  think add get password env code will be better way。

**Comment by ankitpokhrel** [comment]
I see that there is no way to pass redis configs to expired command. Thanks for reporting, will check as soon as I can.

**Comment by ankitpokhrel** [comment]
Fixed by #71, you can now pass path to your config file.

```php
./vendor/bin/tus tus:expired redis --config=<path to your config file>
```

Details [here](https://github.com/ankitpokhrel/tus-php#expiration).
